client/webserver/site/src/html/forms.tmpl

```diff
@@ -1,4 +1,7 @@
 {{define "walletConfigTemplates"}}
+<div id="walletCfgGuide" class="py-2 d-hide">
+  Make sure you follow:  <a id="walletCfgGuideLink" target="_blank" class="py-2 hyperlink">configuration guide</a><br>
```

Comment:
Any text like this unfortunately needs to go in the locales.js like `[[[wallet_follow_the_guide]]]`

Regarding the text itself, why the colon?  Wouldn't it read better as "Make sure you follow the configuration guide."

Reply:
Better with no colon.